identity/keystore_interface.go

```diff
@@ -5,4 +5,7 @@ import "github.com/ethereum/go-ethereum/accounts"
 type keystoreInterface interface {
 	Accounts() []accounts.Account
 	NewAccount(passphrase string) (accounts.Account, error)
+	Find(a accounts.Account) (accounts.Account, error)
```

Comment:
Could You start using `keystore.Find()` in `manager.GetIdentity()` as-well?

Reply:
Yeah, good idea, cheers!

Reply:
should fakeKeystore.Find implement the actual find through fake []Accounts? Otherwise the tests fail, but seems redundant to kind of do it.

Reply:
Fixed.